apps/admin/preview/base/kustomization.yaml

```diff
@@ -3,13 +3,10 @@ kind: Kustomization
 resources:
   - ../../base
   - kube-slack-values.yaml
-  - ../../csi-secret-store-provider-v1.0.1
-  - traefik-values.yaml
```

Comment:
`traefik-values.yaml` is overriding only traefik v1 configuration apparently, so both `../../traefik/traefik.yaml` and `traefik-values.yaml` are being moved to cluster 01 config exclusively. That way it should be easier to remove traefik v1 config completely when moving all new config to (and removing old config from) base.